src/ra_type_traits.h

```diff
@@ -143,7 +143,7 @@ namespace detail {
 template<typename T>
 struct _NODISCARD is_literal_type
     : std::bool_constant<(std::is_scalar_v<T> || std::is_reference_v<T>) ||
-                         (std::is_aggregate_v<T> && std::is_scalar_v<typename T::value_type>) ||
+                         (std::is_aggregate_v<T>) ||
```

Comment:
I was originally trying to make MemBookmark a literal type but doesn't have value_type type member like all STL style  classes.